### ISSUE TITLE
Landscape orientation related issues (text position, button, and scroll related issues)

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#701]
Text gets cut off in landscape orientation
https://github.com/cds-snc/covid-shield-mobile/issues/699

Back button is missing in landscape orientation #698
https://github.com/cds-snc/covid-shield-mobile/issues/698

Some area not scroll-able in landscape on iOS and Android #674
https://github.com/cds-snc/covid-shield-mobile/issues/674 (in an PR https://github.com/cds-snc/covid-shield-mobile/pull/677 awaiting verification)

**Comment by amazingphilippe** [comment]
#698 is an iOS issue. The "Back to COVID Alert" button is provided by the OS, not the app, not the Ontario page.
